lib/ncml_utilities.rb

```diff
@@ -0,0 +1,50 @@
+class NcmlUtilities
```

Comment:
The name of this class perhaps would be better if it was called NcmlFile? since its not just a utitlity class that operates on the file but also holds onto data of the file e.g. @header